examples/tutorial/jupyter/execution/pandas_on_unidist/README.md

```diff
@@ -0,0 +1,47 @@
+# Jupyter notebook examples to run with PandasOnUnidist
+
+Currently, Modin supports `PandasOnUnidist` execution only with MPI backend of [unidist](https://github.com/modin-project/unidist).
+There are some specifics on how to run a jupyter notebook with MPI, namely, you should use `mpiexec` command.
+
+```bash
+mpiexec -n 1 jupyter notebook
+```
+
+**Important**
+
+MPI is not reliable yet to work in interactive environment such as jupyter notebooks. Thus, some things may not work.
+For example, if you are experiencing the error `The kernel appears to have died. It will restart automatically.`,
+you may want to modify `kernel.json` file or create a new one in order to fix the problem.
+
+For simplicity, you can just run `setup_kernel.py` script located in this directory. This will install a new MPI enabled kernel. Otherwise, you can follow the steps below:
```

Comment:
```suggestion
For simplicity, you can just run `setup_kernel.py` script located in this directory. This will install a new MPI enabled kernel,
which you can then select using the dropdown menu in your browser. Otherwise, you can follow the steps below:
```